docroot/modules/custom/open_video_annotation/open_video_annotation.module

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * Plugin for Embedding Open Video Annotation.
+ */
+
+/**
+ * Implements hook_page_attachments().
+ */
+function open_video_annotation_page_attachments(array &$page) {
+  // Adds the OpenVideoAnnotation library to the HTML5 player page.
+  if ($node = \Drupal::routeMatch()->getParameter('node')) {
+    if ($node->getType() == 'article') {
+      $page['#attached']['library'][] = 'open_video_annotation/open_video_annotation.load_player';
+    }
+  }
+}
+
+/**
+ * Implements hook_js_alter().
+ */
+function open_video_annotation_js_alter(&$javascript) {
+  // Open video annotation plugin will take care of video.min.js now.
+  // unset video.min.js file from videojs module .
+  unset($javascript['//vjs.zencdn.net/5.0/video.min.js']);
+}
+
+/**
+ * Implements hook_theme().
+ */
+function open_video_annotation_theme() {
+  return array(
+    'text_link_formatter' => array(
+      'variables' => array('url' => NULL),
+      'template' => 'text-link-formatter',
+    ),
+  );
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function open_video_annotation_preprocess_text_link_formatter(&$variables) {
```

Comment:
This doesn't look right at all. The purpose of this module is to provide a text field formatter from scratch. Don't see a point in implementing the alter hook in the module exposing the formatter plugin itself.